monitor/loadgen/loadgen.go

```diff
@@ -79,5 +81,28 @@ func Start(ctx context.Context, network netconf.Network, ethClients map[uint64]e
 		go selfDelegateForever(ctx, contract, backend, val, period)
 	}
 
+	if xCallerCfg.Enabled {
+		if len(xCallerCfg.ChainIDs) == 0 {
+			return errors.New("xcaller enabled but no chain id pairs specified")
+		}
+
+		xCallerPK, err := eoa.PrivateKey(ctx, network.ID, eoa.RoleXCaller)
+		if err != nil {
+			return errors.Wrap(err, "failed to get RoleXCaller priv key exiting xcall loadgen")
+		}
+
+		backends, err := ethbackend.BackendsFromNetwork(network, endpoints, xCallerPK)
+		if err != nil {
+			return err
+		}
+
+		connector, err := connect.New(ctx, network.ID)
+		if err != nil {
+			return err
+		}
+		xCallerAddr := eoa.MustAddress(network.ID, eoa.RoleXCaller)
+		go xCallForever(ctx, xCallerAddr, period, xCallerCfg.ChainIDs, backends, connector)
+	}
+
```

Comment:
This `Start` function has a lot of stuff in it now. First half about loadgening validator updates. Second part about loadgening xcalls. Maybe put both in their own function